packages/web-frontend/cypress/support/actions.js

```diff
@@ -6,7 +6,7 @@
 import { TEST_USER } from '../constants';
 
 const getTestUserPassword = () => {
-  const password = Cypress.env('TEST_USER_PASSWORD');
+  const password = Cypress.env('CYPRESS_TEST_USER_PASSWORD');
```

Comment:
Did this work for you? It doesn't for me...

[Cypress.env](https://docs.cypress.io/api/cypress-api/env.html#Syntax) loads environment variables prefixed with `CYPRESS_`, but the prefix is omitted when retrieving the variable. We use [cypress-dotenv](https://www.npmjs.com/package/cypress-dotenv) to load those from a `.env` file, but the same convention still applies

Reply:
Thanks for the explanation. My bad, I thought that it didn't work initially but I have re-tested it this morning and it works perfectly with the environment variables set up as you have them. I will put that down to my user error 👍